README.md

```diff
@@ -19,9 +19,9 @@ Community Group Charters:
 
 Working Group Charters:
 
-1. [ActivityPub Maintenance Working Group Charter](https://swicg.github.io/potential-charters/ap-maintenance-wg-charter.html) (ActivityPub-only, maintenance only)
-2. [SocialWeb Maintenance Working Group Charter](https://swicg.github.io/potential-charters/social-web-maintenance-wg-charter.html) (All Social Web Working Group documents, maintenance only)
-3. [IndieWeb Maintenance Working Group Charter](https://swicg.github.io/potential-charters/indieweb-maintenance-wg-charter.html)
+1. [ActivityPub Working Group Charter](https://swicg.github.io/potential-charters/ap-wg-charter.html) (ActivityPub-only, maintenance only)
```

Comment:
```suggestion
1. [ActivityPub Working Group Charter](https://swicg.github.io/potential-charters/ap-wg-charter.html) (ActivityPub-only)
```

Reply:
Strictly speaking, the parenthetical is not part of a proposed name of a WG, so I don't feel the resolution empowered making that change.